### PR TITLE
Correct netcat package name for RHEL >= 7

### PR DESCRIPTION
**Pull request by Reamer**
We should fix this for new redhat Versions
https://access.redhat.com/solutions/1422873

Greetings
Reamer

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -43,6 +40,14 @@
       $libgalera_location           = '/usr/lib64/galera/libgalera_smm.so'
       $additional_packages          = 'rsync'
     }
+    $osr_array = split($::operatingsystemrelease,'[\/\.]')
```
Comment (michaeltchapman) [inline manifests/params.pp]: recent versions of facter have `operatingsystemmajrelease` for this

Comment (Reamer) [inline manifests/params.pp]: that's right, but operatingsystemmajrelease is only available with Facter >= 1.7
I can change that, if you want it.
This code is a partial copy of https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/version.pp#L17

Comment (michaeltchapman) [inline manifests/params.pp]: No problem.

**Comment by michaeltchapman** [comment]
All the tests are failing because facter doesn't populate anything by default. You can fix this by adding the operatingsystemrelease fact to the hash down the bottom of each file, like here: https://github.com/michaeltchapman/puppet-galera/blob/master/spec/classes/galera_init_spec.rb#L104

**Pull request merged**